docs/tutorials/03-publishing-pilets.md

```diff
@@ -19,52 +19,121 @@ We designed Piral to be as simple as possible, while remaining as flexible and p
 
 ## Piral Cloud Services
 
-In this spirit we provide a free[^1] service that you can use for providing the feed service, as well as a way to publish your pilets. This leaves you at the task of creating a Piral instance.
+In this spirit we provide a free[^1] community edition of the service that you can use for providing the **feed service**, as well as a way to publish your pilets. This leaves you at the task of creating a Piral instance.
 
-**Remark:** The specification for a feed service is public. There should be everything you need if you already (or later) want to start building your own feed service. In addition to the publicly available specification we also have published a Node.js sample implementation at GitHub (https://github.com/smapiot/sample-pilet-service). The sample implementation can also be installed or run locally very easily.
+**Remark:** The specification for a feed service is public. There should be everything you need if you already (or later) want to start building your own feed service. In addition to the publicly available specification we also have published a Node.js sample implementation at GitHub (<https://github.com/smapiot/sample-pilet-service>). The sample implementation can also be installed or run locally very easily.
```

Comment:
What are these angle brackets doing there? They also suddenly appear in the URL below. They form invalid markup (and represent both, invalid HTML and invalid Markdown).

Reply:
Removed the angle brackets.